app/controllers/health_check_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class HealthCheckController < ActionController::Base
+  def health
+    response = { version: Kaui.version, status: 'UP' }
```

Comment:
This is the version of the Kaui gem. I think it would be better to show the version of killbill-admin-ui-standalone, the Rails app itself, as this is what the user installs.

I believe this version is only in the pom.xml (https://github.com/killbill/killbill-admin-ui-standalone/blob/6a3a028c2d06e07724aa7d03187348154f81880d/pom.xml#L24) today, so we need to change the release process a bit to do something like https://github.com/killbill/killbill-admin-ui/blob/841e6fac9df751fab8ae4543dabb2e2c91e4df77/.github/workflows/release.yml#L33C23-L33C31, in order to have that version at runtime.